README.md

```diff
@@ -1 +1,152 @@
-# document-management-system
\ No newline at end of file
+
+# Document Management System API
+
+[![Build Status](https://travis-ci.org/andela-cnwankwo/document-management-system.svg?branch=develop)](https://travis-ci.org/andela-cnwankwo/document-management-system)  [![Test Coverage](https://codeclimate.com/github/andela-cnwankwo/document-management-system/badges/coverage.svg)](https://codeclimate.com/github/andela-cnwankwo/document-management-system/coverage)   [![Code Climate](https://codeclimate.com/github/andela-cnwankwo/document-management-system/badges/gpa.svg)](https://codeclimate.com/github/andela-cnwankwo/document-management-system)  
+
+
+## Introduction
+
+The system manages documents, users and user roles. Each document defines access rights; the document defines which roles can access it. 
+Each document specifies the date it was published.
+
+## Technology
+
+This application is built using NodeJS, Express and Postgres
+
+## Features
+This application has the following features:
+
+### Users:
+A created user will have a role, either an admin or a regular user by default.
+- A Regular User can: 
+    - Create an account
+    - Login
+    - Create a document
+    - Limit access to a document by specifying an access group `i.e. public, private or role`.
+    - View public documents created by other users.
+    - View documents created by his access group with access level set as `role`.
+    - Edit his record.
+    - Search a users public documents.
+    - View `public` and `role` access level documents of other regular users.
+    - Logout.
+    - Delete his details.
+
+- In addition to the general user functions, an admin user can:
+    - View all users.
+    - View all created documents.
+    - Delete any user.
+    - Update any user's record.
+    - Create a new role.
+    - View all created roles.
+    - Search for any user.
+
+### Documents:
+Documents are created to have various properties.
+They include:
+- Published date
+- Title
+- Content
+- Access (`private, public or role`)
+
+### Roles:
+Roles can also be created, the default roles are `admin` and `regular`
+
+### Authentication:
+Users are authentcated and validated using JSON web token (JWT).
+By generating a token on registration and login, API endpoints and documents are protected from unauthorised access.
+Requests to protected routes are validated using the generated token.
+
+## Installation
+- *Steps*:
+    - Clone this repository by running the command on the terminal: `git clone https://github.com/andela-cnwankwo/document-management-system.git`
+    - Navigate to the project folder and install the packages/modules using: `npm install`
+    - Start the server by running: `npm start`
+    - Use an API testing platform ( `e.g. Postman `) to test the endpoints ( `see endpoints below` )
+
```

Comment:
In your `README.md`, I see you handled this in the **Test** section, but you might also want to add more steps to host this application locally.